cpp/include/cuvs/neighbors/ivf_flat.hpp

```diff
@@ -17,13 +17,18 @@
 #pragma once
 
 #include "ann_types.hpp"
+#include "ivf_list.hpp"
 #include <raft/neighbors/ivf_flat_types.hpp>
```

Comment:
Now that we are only exposing runtime APIs publicly, we don't need to separate out the `*_types.hpp` anymore.